CMake/PythonDocstrings.cmake

```diff
@@ -55,11 +56,24 @@ endif()
 get_filename_component(_docstrings_path ${CMAKE_CURRENT_LIST_FILE} DIRECTORY)
 set(_docstrings_path ${_docstrings_path}/..)
 
-function(DOCSTRINGS BOOST_PYTHON_SOURCES CPP_HEADERS)
+function(DOCSTRINGS DOCSTRINGS_INCLUDE_PATH BOOST_PYTHON_SOURCES CPP_HEADERS)
```

Comment:
I'd put the include path as the last argument and rename it to EXTRA_INCLUDE_PATH or simply INCLUDE_PATH, DOCSTRINGS is redundant if it becomes a parameter.